ansible-ipi-install/README.md

```diff
@@ -298,15 +298,16 @@ pullsecret=""
 # The hardware_profile is used by the baremetal operator to match the hardware discovered on the host
 # See https://github.com/metal3-io/baremetal-operator/blob/master/docs/api.md#baremetalhost-status
 # ipmi_port is optional for each host. 623 is the common default used if omitted
+# poweroff=true indicates the playbook should power off the nodes before deploying OCP
```

Comment:
Since I see this as a specialty case,  I think changes should only be needed for that specialty case.  In other words, let not make every one have to make chagnes, the default should power off the nodes, and in case where you dont want to power off nodes, you can optionally add the poweroff=false

Reply:
done